### DISCUSSION TITLE
Stupid Question: How do I create a "tag" when creating a release?

### DISCUSSION THREAD

**Discussion by Jark5455** [community/community#41792] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

The instructions on the docs say to click on "create new tag" in the release page however whenever I do that the dropdown simply disappears and does nothing :| I have tried on multiple browsers and I think there's a user error going on here. [this](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) is the specific doc I am referring to.

**Comment by radar-blip** [comment, marked as answer]
After clicking on create new tag, (and typing a new tag name) the new tag name should be suggested for you right below that. 
Just click on the suggested tag name.

![tags](https://user-images.githubusercontent.com/39196767/207763697-5988a31d-7706-4e5c-ac32-2b68f8b7a9e6.png)

If you have existing tags, you can try editing the release.

![tags2](https://user-images.githubusercontent.com/39196767/207765870-84c78728-68f9-4ad6-97c9-f9f0ec4dfa85.png)

**Comment by ipupiara** [comment]
stupid question you wrote !!!  that is by far not a stupid question. I am searching on this also quite some times.  The only worst stupid thing I see is the github web pages. they have never heard about user friendly applications.....  and their description help you only if you yet kmow how it works -- usually hopefully old info pages that where maybe once valid.
